.github/pull_request_assignment.yml

```diff
@@ -243,3 +243,4 @@
       - Wzb123456789
       - kazrael2119
       - v-xuto
+      - rkmanda
```

Comment:
@raych1 why is this under "**" as opposed to the "resource-manager" section above?

Reply:
We should also make sure that the common-types\resource-manager is included in this pattern and remove the generic common-types path above.